Userland/Applications/FileManager/PropertiesWindow.cpp

```diff
@@ -61,10 +70,16 @@ PropertiesWindow::PropertiesWindow(DeprecatedString const& path, bool disable_re
         m_apply_button->set_enabled(m_name_dirty || m_permissions_dirty);
     };
 
+    auto location = general_tab.find_descendant_of_type_named<GUI::LinkLabel>("location");
+    location->set_text(m_path);
+    location->on_click = [this] {
+        Desktop::Launcher::open(URL::create_with_file_scheme(m_parent_path, m_name));
+    };
+
     struct stat st;
-    if (lstat(path.characters(), &st)) {
+    if (lstat(m_path.characters(), &st)) {
         perror("stat");
-        return;
+        return Error::from_errno(errno);
     }
```

Comment:
A more complete way to do to this refactor would be to say `auto st = TRY(Core::System::lstat(m_path));` instead of keeping the perror and manually returning errno.